tests/helper.py

```diff
@@ -12,6 +12,9 @@ def assert_dtype_allclose(dpnp_arr, numpy_arr, check_type=True):
     Assert DPNP and NumPy array based on maximum dtype resolution of input arrays
     for floating and complex types.
     For other dtypes the assertion is based on exact matching of the arrays.
+    When 'check_type' True (default), it asserts equal dtypes for exact types
+    and either equal dtypes or kinds for inexact types, depending on the 64-bit precision
+    support of the device on which `dpnp_arr` is created.
```

Comment:
```suggestion
    When 'check_type' is True (default), it asserts dtypes of input arrays for exact matching
    of either dtypes or kinds. The last is in case of inexact dtypes and support of fp64 asspect
    by the device on which `dpnp_arr` was created.
```